### DISCUSSION TITLE
Readibility poor since last update

### DISCUSSION THREAD

**Discussion by DianaLaa** [gitextensions/gitextensions#9367] [Q&A]
I updated this morning. The update apparently involves color scheme changes. This has negatively affected readability (very much). Greens and reds on a yellow background is not suitable for vision impaired and colorblind users.

Can I revert the color scheme myself?

- Git Extensions 3.5.1.12196
- Build 9099a1e76d01dbbd6896d6b026027db0b68fba96
- Git 2.30.0.windows.1
- Microsoft Windows NT 10.0.19041.0
- .NET Framework 4.8.4390.0
- DPI 120dpi (125% scaling)

**Comment by RussKie** [comment]
Colour schemes are here:
![image](https://user-images.githubusercontent.com/4403806/125298198-48f5b480-e36b-11eb-9866-afc957f8a7e5.png)

**Comment by DianaLaa** [comment]
Thanks for the suggestion. I've changed to the light, colorblind theme, but that still does not give clear, crips colors and good contrast. This is yellow with murky blue/red. How do I get the old theme back? White background, bright red and bright green tag/branches?

**Comment by DianaLaa** [comment]
Can I download the old theme somewhere to import it again?

**Reply by mstv** [reply]
Not completely due to the background (refer to [my reply below](https://github.com/gitextensions/gitextensions/discussions/9367#discussioncomment-995832)).

In order to get maximum contrast and invariant background, you could disable `Highlight authored revisions` and perhaps `Draw alternate background` for the time being:

![image](https://user-images.githubusercontent.com/36601201/125354101-af76d480-e363-11eb-9f3d-19ec7b2a68b2.png)

Regarding the colors, you can create your own theme file (and inherit the other colors from the most suitable theme). Or ask @jrfnl.

**Reply by gerhardol** [reply]
The changes if I recall correctly:
* adding color blind adaptions
* switch red/green for local/remote branches (to align with git-log)
* (In 3.5.0 the colors for green/red were a little more pastel, changed back in 3.5.1)
* remove the background for labels
* adding a few paddings etc

So there is no old colors to import.
I expect that you react on the branch/tag labels, can you confirm with 3.4? (use portable)

**Reply by jrfnl** [reply]
Thanks for the ping @mstv. I reported these issues before in #9111 (low contrast) and #9113 (reversed colours). The discussions in those issues may give some additional suggestions.

I've been using a custom colour scheme since and will upload that. Maybe it can help @DianaKoenraadt or someone else.
It doesn't bring back the old colour scheme completely, but it matches it as closely as I could get it.

[juliette.zip](https://github.com/gitextensions/gitextensions/files/6804696/juliette.zip)

To use it:
* Unzip the file (and possible rename it to a name which suits you).
* Place it in the `Users\%USERNAME%\AppData\Roaming\GitExtensions\GitExtensions\Themes` directory. You may need to create this directory if it doesn't exist yet.
* Then in settings, go to Appearance -> Colors and choose the custom theme under Theme.
* Restart GitExtensions.

Hope this helps.

**Reply by DianaLaa** [reply]
Thanks for the sugegstions. I was able to remove the yellow background. But the tags/branches still don't have enough contrast for me. I've imported juliette's theme, but that doesn't have solid tags/branch rectangles either. Readibility is still poor for me.

**Comment by andyraf** [comment]
I also feel the new color scheme is unpleasant to work with. I think the change was that in the prior design the text had a very light shaded pastel background matching the color of the text, while in the new scheme the background is transparent and just shows through to whatever color is behind the text. Is there a way in the theme to specify the text background color or shading?
![gitextensions](https://user-images.githubusercontent.com/11358117/125351707-423c5280-e315-11eb-9d95-a7b32448bd86.jpg)

**Reply by mstv** [reply]
The new style is even more varying than in your screenshot.
The background filling has been replaced with a frame. Some find it cool, I cannot follow.

New | Old
-|-
![image](https://user-images.githubusercontent.com/36601201/125353539-fd3f0d00-e362-11eb-9080-fcce76b0797e.png)|![image](https://user-images.githubusercontent.com/36601201/125355734-a8e95c80-e365-11eb-88ce-0956844eb578.png)
